Pokerole 1e/sheet.json

```
@@ -0,0 +1,28 @@
+{
+	"html": "Pokerole.html",
+	"css": "Pokerole.css",
+	"authors": "Chevron Allen",
+	"roll20userid": "2130808",
+	"preview": "Pokerole.jpg",
```

```suggestion
	"preview": "Pokeroll.jpg",
```
the preview image names doesn't match.